hypha/apply/determinations/templates/determinations/determination_detail.html

```diff
@@ -4,42 +4,43 @@
 {% block title %}{% trans "Determination for" %} {{ determination.submission.title }}{% endblock %}
 
 {% block content %}
-<div class="admin-bar">
-    <div class="admin-bar__inner">
-        <a class="simplified__submissions-link" href="{{ determination.submission.get_absolute_url }}">
-            {% trans "Back to submission" %}
-        </a>
-        <h2 class="heading heading--no-margin">{% trans "Determination" %} {% if determination.is_draft %}[{% trans "DRAFT" %}]{% endif %}
-        </h2>
-        <h5>{% trans "For" %}: {{ determination.submission.title }}</h5>
```

Comment:
I think the submission title is useful here. We have it on reviews detail page as well. On reviews it includes applicant name and submission date as well Do not know if that is needed, butI think they should look the same.

Without the submission detail you do not know what submission the determination is for unless you click the "Back to submission" link.